.github/workflows/go.yml

```
@@ -35,17 +35,24 @@ jobs:
         mkdir --parents /home/runner/go/src/github.com/vertgenlab/ &&
         cp -r ../gonomics /home/runner/go/src/github.com/vertgenlab/ &&
         cd /home/runner/go/src/github.com/vertgenlab/gonomics &&
+        go install ./... &&
         go test ./...
         
   format:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
```

I would just upgrade to v3, reference: [actions/checkout#checkout-v3](https://github.com/actions/checkout#checkout-v3)
```suggestion
      - uses: actions/checkout@v3
```